pegasus/sites.v3/code.org/public/css/dance-landing/dance.css

```diff
@@ -2,6 +2,55 @@
   font-size: 20px;
 }
 
+.img-dance-stars {
+  position: absolute;
+  top: 10px;
+  left: 70px;
+  width: 330px;
+  padding: 20px 10px 10px;
+}
+
+.desktop-only, .tablet-only, .mobile-only {
+  display: none;
+}
+
+.starwars-tutorial.dance-tutorial {
```

Comment:
does this need to reference the `starwars-tutorial` class? or can it just be `.dance-tutorial`?

Reply:
Good question.  Per our in-person discussion, this was the only way to ensure the style was being applied to dance tutorial section due to css specificity.